pctasks/dataset/pctasks/dataset/workflow.py

```diff
@@ -208,6 +209,17 @@ def create_process_items_workflow(
         target_environment=target,
     )
 
+    if is_update_workflow:
```

Comment:
@lossyrob this is the relevant portion to review. https://github.com/microsoft/planetary-computer-tasks/pull/72/files#diff-dd254cd830815667d1f234ab1514bb7bc99486fc79cc49d2af0b5505d21207f4R286 has the docs.